docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -985,7 +985,7 @@
     "\n",
     "obj_QUBO = (\n",
     "    obj.to_qubo()\n",
-    ")  # converts our objective function into a QUBOWe then set up the client connection in the following cell."
+    ")  # Converts our objective function into a QUBOWe then set up the client connection in the following cell."
```

Comment:
(precedes this PR) is this supposed to be:
```suggestion
    ")  # Converts our objective function into a QUBO. We then set up the client connection in the following cell."
```
?

Reply:
Yes, fixed now.